src/jshint.js

```diff
@@ -6296,7 +6296,7 @@ var JSHINT = (function() {
     } else {
       JSHINT.errors = [];
       JSHINT.internals = [];
-      JSHINT.blacklist = {};
+      JSHINT.denylist = {};
```

Comment:
This is a breaking change. We're not ready for a major release, but what we *can* do in the mean time is label `JSHINT.denylist` as "deprecated" and offer the more inclusive API as an alternative. So, `JSHINT.blacklist = JSHINT.denylist;`.